ev-dev-tools/src/ev_cli/ev.py

```diff
@@ -107,16 +107,17 @@ def generate_tmpl_data_for_module(module, module_def):
         })
 
     requires = []
-    for impl, impl_info in module_def.get('requires', {}).items():
-        is_optional = impl.startswith('optional:')
-        if is_optional:
-            impl = impl[9:]
+    for requirement_id, req_info in module_def.get('requires', {}).items():
+        # min_connections=1 and max_connections=1 is the default if not provided otherwise (see manifest meta schema)
+        is_vector = not (
+            ('min_connections' not in req_info or req_info['min_connections'] == 1) and
```

Comment:
Sorry for being a wisenheimer here (I guess I'm not allowed to say smart-ass anymore due to inclusive language restrictions), but this could be written as
req_info.get('min_connections', 1) == 1
but maybe we want to do the same here as in the everest framework, that is patching the json with the meta schema

Reply:
yeah, patching with the meta schema would be the best, that one slipped through.
The old python framework has code for this (unfortunately the python json-schema lib we are using does not include such patching code already)